src/test/java/org/folio/service/CirculationRequestsRetrieverTest.java

```diff
@@ -0,0 +1,186 @@
+package org.folio.service;
+
+import static org.folio.TestConfig.autowireDependencies;
```

Comment:
I didn't want to confuse you, but we should move to Intellij idea default import style. The old files it is better to stick old style, for the new files, it is better to move to new style. I saw many pr, where developer are using default java import style

just open settings and change to Project that will respect to existing import style if there is, otherwise it will use default import style

Reply:
Got it, thanks, I believe it should be correct now